httpcore/_utils.py

```diff
@@ -47,3 +48,10 @@ def trace(message: str, *args: typing.Any, **kwargs: typing.Any) -> None:
     logger.trace = trace  # type: ignore
 
     return typing.cast(Logger, logger)
+
+
+def url_to_origin(url: URL) -> Origin:
+    scheme, host, explicit_port = url[:3]
+    default_port = {b'http': 80, b'https': 443}[scheme]
```

Comment:
I was curious why Black didn't cough up on this usage of single quotes, only to realize it's not run in `scripts/check`.

It's been a while since I've been around here but I assume it (and the isort check) was removed on purpose from there?